llama_hub/tools/python_file/README.md

```diff
@@ -0,0 +1,30 @@
+# Python File Tool
+
+This tool loads a python file and extracts function names, arguments and descriptions automatically. This tool is particular useful for automatically creating custom Tool Specs when you already have well documented python functions. 
+
+## Usage
+
+This tool has more extensive example usage documented in a Jupyter notebook [here](https://github.com/emptycrown/llama-hub/tree/main/llama_hub/tools/notebooks/create_a_tool.ipynb)
+
+Here's an example usage of the PythonFileToolSpec.
+
+```python
+from llama_hub.tools.openapi.base import PythonFileToolSpec
```

Comment:
import is wrong

Reply:
not sure If I had too much or not enough coffee yesterday 😮‍💨